.github/workflows/publish-azureagent.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and publish the azure-pipelines-azureagent image
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'azure-pipelines-azureagent/*'
+  schedule:
+    # rebuild image every Sunday
+    - cron: '43 3 * * 0'
+
+jobs:
+  build:
+    name: Build and publish image for azure-pipelines-azureagent
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    
+    - name: Determine latest release of VSTS Agent
+      id: getvstsagentlatest
+      run: |
+          release=$(curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/microsoft/azure-pipelines-agent/releases/latest | jq -r .name)
+          echo "vstsagentrelease=$release" >> $GITHUB_ENV
```

Comment:
can we also add a regular echo so the version is clearly visible in the output?

Reply:
done